### ISSUE TITLE
qustion for loss function

### ISSUE THREAD

**Issue by LuoJiaji** [Zardinality/WGAN-tensorflow#10]
In WGAN.ipynb I find the loss function is  tf.reduce_mean(fake_logit - true_logit)，I don't understand that. I find the loss function in other code is  tf.reduce_mean(self.d) - tf.reduce_mean(self.d_). I'm wondering which is correct?

**Comment by Zardinality** [comment]
Please refer to the fourth note in the readme, I think it is clear enough.

**Comment by LuoJiaji** [comment]
Thanks a lot. I understand
